ssl_cert_tracker/tasks.py

```diff
@@ -29,7 +29,7 @@
 logging.basicConfig(filename='p_soc_auto.log', level=logging.DEBUG)
 
 
-@shared_task(rate_limit='30/s', queue='nmap')
+@shared_task(rate_limit='0.5/s', queue='nmap')
```

Comment:
rate_limit means # of tasks per second in unit of time
In above case it is 0.5 per second and if we have 100 tasks to execute in this case then we have to execute them in total of 50 seconds (to fetch the xml data, parse the data, and populate models) which is very risky i

Reply:
i think your definition of rate_limit is not correct. 0.5/s means 1 task every 2 seconds. 100 tasks would execute in 200 seconds

Reply:
Still Running one task every 2 seconds means assuming we need to have the result back in less than 2 seconds before putting another tasks on the queue.
This could be an issue 
We need to find out roughly how many tasks in total indented to be processed during a an interval and distribute them accordingly